### PR TITLE
JEFF_Validation.csv

### PR DESCRIPTION
**Pull request by julesdamasta**
# Validate [{JEFF}](https://solscan.io/token/FkCaFsprX7gySagFoQPHNbe9MRkjrvh21cokJo6C1e2T}) 

## Attestations (Please provide links):
- Tweet from your Twitter Account attesting the Mint address, tagging [@JupiterExchange](https://twitter.com/TheJeffcoinSOL/status/1679172618392330240?s=20)
- Coingecko/ CMC URL (If available): N/A

## Validation (Please check off boxes):
- [y] The metadata provided in the PR matches what is on-chain (Mandatory)
- [y] Does not duplicate the symbol of another token on Jupiter's strict list (If not, review will be delayed)
- [n] Is Listed on Coingecko / CMC (Optional, but helpful for reviewers)  

## Acknowledgement (Please check off boxes)
- [y ] My change matches the format in the file (no spaces between fields).
- [y ] My token is already live and trading on Jupiter.
- [y ] !!! I read the README section on Community-Driven Validation and understand this PR will be only be reviewed when there is community support on Twitter.

### CODE REVIEW

**Comment by Timoon21** [comment]
Can you add a Tweet from your Twitter Account attesting the Mint address, tagging [@JupiterExchange](https://twitter.com/JupiterExchange) and showing community support ? Not just your twitter account.

I'll leave the PR open for now and review it again in a few days.

**Comment by julesdamasta** [comment]
As requested... https://twitter.com/TheJeffcoinSOL/status/1679172618392330240?s=20

File: validated-tokens.csv [hunk 1]
```diff
@@ -744,3 +744,4 @@ BULL,BULL,BULLa6g9e5UCuTXC5Z3Cf7s7CgvJhnJfY71DwipSmF8w,9,https://shdw-drive.gene
 Pink,PINK,3AMtw8m3pZnrGLmGqQcJnHSQSaegtZGBMpon74EVZVQ3,6,https://arweave.net/_ZVGjmZaB-1kxDkJzamQFo-kxJD0Kl0NhVx8dowP5zE,true
 REJECT Token,REJECT,4F2yutcbkabE5MJoDvrDLa5U2re5HPABSCVKA7vqrKcH,2,https://arweave.net/ijxRvhTwJRWTanYUI-j2DwPzOkpqRbB3y-qSuuvQqhQ,true
 Ethrix,ETX,9DZ58i5vAfk3JaFVYezYzhrVht7j8McZsUbuTcDiSbrP,5,https://arweave.net/iOND5JLSsq_0dka0SnzTsDCl83hu0_-sOQ2fDbQDuTQ,true
+JEFF, JEFF,FkCaFsprX7gySagFoQPHNbe9MRkjrvh21cokJo6C1e2T,6,https://pbs.twimg.com/profile_images/1674243670747357185/mxqfTQkJ_400x400.jpg,true
```
Comment (Timoon21) [inline validated-tokens.csv:747]:
```suggestion
JEFF, JEFF,FkCaFsprX7gySagFoQPHNbe9MRkjrvh21cokJo6C1e2T,6,https://arweave.net/Pi-s2vzZT8pfYVvFbNjgC0kSrW45PXb1_VPb5HoHuyI,true
```

**Review by Timoon21 — CHANGES_REQUESTED** [review]

**Review by Timoon21 — APPROVED** [review]
- [x] On-chain metadata matches the PR
- [x] Token address matches the tweet
- [x] No duplication on Jupiter's strict list
- [x] Community validation

**Comment by julesdamasta** [comment]
ty!

**Pull request merged**